common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
+                            "and clearly marked as paid content."
+                        ),
                         "is_checked": False,
-                        "action_url": "https://edge.edx.org/courses/edX/edX101/How_to_Create_an_edX_Course/about",
-                        "action_text": _("Register for edX 101"),
-                        "action_external": True,
+                        "action_url": "CourseOutline",
+                        "action_text": _("View Course Outline"),
+                        "action_external": False,
                     },
                     {
-                        "short_description": _("Download the Studio Documentation"),
-                        "long_description": _("Download the searchable Studio reference documentation in PDF form."),
+                        "short_description": _("Clearly indicate off-platform content"),
+                        "long_description": _(
+                            "If you've linked learners to third-party tools outside of the edX course, ensure that you've"
+                            "clearly communicated that this linked content is outside of the edX platform and any terms of"
+                            "service of the third-party service apply."
+                        ),
                         "is_checked": False,
-                        "action_url": "http://files.edx.org/Getting_Started_with_Studio.pdf",
-                        "action_text": _("Download Documentation"),
-                        "action_external": True,
+                        "action_url": "CourseOutline",
+                        "action_text": _("View Course Outline"),
+                        "action_external": False,
                     },
                 ],
             },
             {
-                "short_description": _("Draft Your Course About Page"),
+                "short_description": _("Learner Engagement"),
                 "items": [
                     {
-                        "short_description": _("Draft a Course Description"),
+                        "short_description": _("Send welcome email"),
+                        "long_description": _(
+                            "You agree to send a welcome email to learners on the course start date."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Send emails throughout course run"),
+                        "long_description": _(
+                            "You agree to send paced emails (recommended: once a week) to students throughout the course run" 
+                            "to communicate when new content is released, to update on course events, etc."
```

Comment:
At intervals throughout the course (once a week is recommended), you agree to send email messages to learners to communicate the release of new content, provide updates on course events, and so on.